calebwagner/button-the-clown
# rendered html to main.js

I create an index.html module, rendered the soon to be html in the main.js module, created the format for the html in the ClownService.js module.

Steps to check code:
1. git fetch --all
2. git checkout cjw-main-module
3. code .
4. check main.js for rendered html
5. check for index.html module
6. checkout in ClownsService.js module for html
